src/main/java/com/epam/service/PersonService.java

```diff
@@ -0,0 +1,34 @@
+package com.epam.service;
+
+
+
+import com.epam.model.Person;
+import com.epam.repository.PersonDAO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class PersonService {
+
+    private final PersonDAO personDAO;
+    @Autowired
+    public PersonService(PersonDAO personDAO) {
+        this.personDAO = personDAO;
+    }
+
+    public void printPersonsNames(){
+        List<Person> persons = personDAO.getPersons();
+        persons.forEach(person -> System.out.println(person.getName()));
+    }
+
+    public Person getPerson(String name){
```

Comment:
Read about DRY principle: you have already had good method above, try to use it here

Reply:
What will be if we pass null as an argument?